src/domain/event/eventForm/EventForm.tsx

```diff
@@ -170,4 +184,96 @@ const EventForm = () => {
   );
 };
 
+const ImageSelectedForm: React.FC<{
+  image: string;
```

Comment:
Could be moved to own file. ImageSelectedForm is little bit misleading name because this component is not form

Reply:
Good point. Any suggestions for a better name? ImageSelectedFormPart?

Reply:
That's ok to me. Let's try to use same convention in the future as well to keep the codebase consistent

Reply:
👍